loader/vk_loader_platform.h

```diff
@@ -396,23 +421,6 @@ static char *loader_platform_get_proc_address_error(const char *name) {
 // Threads:
 typedef HANDLE loader_platform_thread;
 
-// __declspec(thread) is not supported by MinGW compiler (ignored with warning or
-//                    cause error depending on compiler switches)
-//
-// __thread should be used instead
-//
-// __MINGW32__ defined for both 32 and 64 bit MinGW compilers, so it is enough to
-// detect any (32 or 64) flavor of MinGW compiler.
-//
-// @note __GNUC__ could be used as a more generic way to detect _any_
-//       GCC[-compatible] compiler on Windows, but this fix was tested
-//       only with MinGW, so keep it explicit at the moment.
-#if defined(__MINGW32__)
```

Comment:
Why'd you remove these?  I don't see them defined anywhere else in the new source.

Reply:
Because TLS was being used for parameter passing, not solving specific threading problems. Passing in the required parameters to the functions instead of storing a pointer in TLS makes the code more straightforward and requires less compiler specific workarounds, like the stuff mentioned here.